redis/internal/redisprom/pools.go

```diff
@@ -0,0 +1,43 @@
+package redisprom
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const RedisPoolLabel = "redis_pool"
```

Comment:
We define it as a label here but is it never set with a value when emitting metrics?

Reply:
is it on all the metrics defined below in this file.

or am I misunderstanding your comment?

Reply:
```suggestion
const redisPoolLabel = "redis_pool"
```
Does it need to be exported then, can we just define it locally here?

Reply:
done